client/extensions/woocommerce/app/settings/shipping/shipping-zone-dialog.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import clone from 'lodash/clone';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Dialog from 'components/dialog';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormTextInput from 'components/forms/form-text-input';
+import FreeShippingMethod from './shipping-methods/free-shipping';
+import LocalPickupMethod from './shipping-methods/local-pickup';
+import TokenField from 'components/token-field';
+
+class ShippingZoneDialog extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.freeShippingDefaults = {
+			methodId: 'free',
+			everyone: true,
+			minSpend: 0
+		};
+
+		this.localPickupDefaults = {
+			methodId: 'local',
+			price: 0,
+			taxable: false
+		};
+
+		this.state = {
+			location: [],
+			shippingMethods: [ clone( this.freeShippingDefaults ) ] };
```

Comment:
You can avoid importing `clone` and use the spread operator here instead.

`shippingMethods: [ { ...freeShippingDefaults } ],`.